app/cdap/components/WrangleHome/Components/OngoingDataExploration/index.tsx

```diff
@@ -26,7 +26,7 @@ import { forkJoin } from 'rxjs/observable/forkJoin';
 
 const OngoingDataExploration = (props) => {
   const [ongoingExpDatas, setOngoingExpDatas] = useState<any>([]);
-  const [finalArray, setFinalArray] = useState([]);
+  let [finalArray, setFinalArray] = useState([]);
```

Comment:
Prefer to keep this as `const` to reflect that it should match what came from `state`. It's better to create a new variable for the results of the `filter` on line 104.

Reply:
done